classes/core-image-pelux.bbclass

```diff
@@ -7,6 +7,11 @@ inherit core-image
 
 IMAGE_FEATURES += "ssh-server-openssh package-management debug-tweaks"
 
+# Complete ssh-server-openssh with a sftp server
+IMAGE_INSTALL += "\
+    openssh-sftp-server \
```

Comment:
Shouldn't the `openssh` features be included only in the dev image?

Reply:
What you said ;)